src/overlays/actors/ovl_En_Grasshopper/z_en_grasshopper.h

```diff
@@ -6,6 +6,8 @@
 
 #define EN_GRASSHOPPER_GET_TYPE(thisx) ((thisx)->params)
 
+#define EN_GRASSHOPPER_PARAMS(type) (type)
```

Comment:
When the params only take a single value, do we still want to have a macro like this? For example, ClearTag currently just uses the `ClearTagType` directly when spawning via `Actor_Spawn`? I suppose it would be consistent to have the trivial PARAMS enum

Reply:
For reference, this is a more general question outside the scope of this PR, so we can always discuss it later